src/server/server_family.cc

```diff
@@ -1932,6 +1940,20 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
   if (should_enter("PERSISTENCE", true)) {
     auto save_info = GetLastSaveInfo();
 
+    size_t current = m.current_save_stats.keys_processed;
+    size_t total = m.current_save_stats.keys_total;
+    double perc = 0;
+    if (has_save_controller && total != 0) {
+      perc = (static_cast<double>(current) / total) * 100;
+    } else {
+      current = 0;
+      total = 0;
+    }
+
+    append("current_fork_perc", perc);
```

Comment:
I believe we can change the name here, use different one than redis as we dont have a fork

Reply:
I agree I just opted in for consistency instead of adding a different name. I will rename accordingly